tags/app.tag

```diff
@@ -17,19 +24,23 @@
   <contact-list></contact-list>
 
   <script>
-  // this.data = [
-  //   {id="", title="appsContainer", body="click the dashbord link above."},
-  //   {id="1", title="createAppContainer", body="click the new app link above."},
-  //   {id="2", title="detailAppContainer", body="click the apps detail link above."},
-  // ]
-  // console.log(this.data)
-  //
-  // this.page = this.data[0]
-  //
-  // riot.route(function(id){
-  //   this.page = this.data.filter(function(r){ return r.id == id})[0] || {}
-  //   this.update()
-  // })
+  var self = this
+  this.data = [
+    {id: "", title: "appsContainer", body: "click the dashbord link above."},
+    {id: "1", title: "createAppContainer", body: "click the new app link above."},
+    {id: "2", title: "detailAppContainer", body: "click the apps detail link above."}
+  ]
+
+  this.page = this.data[0]
+
+  riot.route(function(id){
+    self.page = self.data.filter(function(r){ return r.id == id})[0] || {
+      title: 'Not found',
+      body: 'Specified is not found'
+    }
+    self.update()
```

Comment:
here is where we need to use self now, because `this` refers now to `function(id){ ... }`, and no longer to the `<app>` tag context